### ISSUE TITLE
OSX: The window moves unintentionally between several desktops.

### ISSUE THREAD

**Issue by jsiedentop** [deiucanta/chatpad#115]
First of all, I would like to thank you for your great work! A truly outstanding app. If only there wasn't this one super annoying thing:

The window is quite clingy and runs after me like a one-year-old child and can't spend a moment alone on a desktop to concentrate on my work. Is this desired behavior? After all, the app is only one year old, but I honestly don't want to wait until it reaches kindergarten age.

Observed behavior: 
I work under OSX with several desktops. If I switch from one desktop to another, the window always jumps to the new desktop. If the window was previously in focus, it is now in the foreground. If it was not in focus, it is in the foreground for 1 second and then disappears into the background.

Expected behavior:
If I open the program on a desktop, the window should remain on this desktop. Just like any other program does.

**Comment by jsiedentop** [comment]
Since i did not find a reference to the todesktop library in the repository, i assume that you created the app using their no code approach. I skimmed the documentation and it mentions that you can customize the window behavior. Is there any configuration file for toDesktop?

**Comment by Supernova3339** [comment]
No, but I'm assuming this is due to you having "always on top" enabled.
